getting-started/proc-configuring-keycloak-with-oidc.adoc

```diff
@@ -0,0 +1,83 @@
+[id="proc-configuring-keycloak-with-oidc_{context}"]
+
+= Configuring Keycloak with the OIDC authentication provider
+
+//You can use Keycloak to authenticate with {product-short} using the OIDC authentication provider. 
+
+{product} includes an OIDC authentication provider that can authenticate users by using Keycloak.
+
+
+
+[IMPORTANT]
+====
+The user that you create in Keycloak must also be available in the {product-short} catalog.
+====
+////
+.Prerequistes
+. You configured {product-short} with a custom config map and secret.
+
+.Procedure
+////
+
+.Procedure
+. In Keycloak, create a new realm, for example `RHDH`.
+. Add a new user.
+
+*Username*:: Username for the user, for example: `rhdhuser`
+*Email*:: Email address of the user.
+*First name*:: First name of the user.
+*Last name*:: Last name of the user.
+*Email verified*:: Toggle to *On*.
+
+. Enter a *Password* for the user that you have created and toggle *Temporary* to *Off*.
+
+. Create a new Client ID, for example, `RHDH`.
+
+*Client authentication*:: Toggle to *On*.
+*Valid redirect URIs*:: Set to the OIDC handler URL, for example, `https://<RHDH_URL>/api/auth/oidc/handler/frame`.
+
+. Navigate to the *Credentials* tab and copy the *Client secret*.
+
+. Save the Client ID and the Client Secret for the next step.
+
+. In {product-short}, add your Keycloak credentials in your {product-short} secrets.
+
+.. Edit your Developer Hub secrets, such as secrets-rhdh.
```

Comment:
```suggestion
.. Edit your {product-short} secrets, such as `secrets-rhdh`.
```

Is `secrets-rhdh` a file or a parameter? Maybe we should specify that for clarity (i.e. Edit your {product-short} secrets [file/parameter], such as...)

Reply:
Elsewhere in our docs we just refer to 'secrets', I believe it is a term commonly used in OCP without specifically saying how it is stored e.g. in a file or as part of a config map.